vault/dynamic_system_view.go

```diff
@@ -65,7 +65,9 @@ func (e extendedSystemViewImpl) SudoPrivilege(ctx context.Context, path string,
 	// Resolve the token policy
 	te, err := e.core.tokenStore.Lookup(ctx, token)
 	if err != nil {
-		e.core.logger.Error("failed to lookup token", "error", err)
+		if !e.core.IsDRSecondary() {
```

Comment:
Are we sure that we need this check here? `SudoPrivilege` is called by `handleCreateCommon`, which should not be called by DR secondaries, right?

Reply:
i'm not sure. i don't have a stack trace, only the error which is the same in both places

in an ideal world, nothing that expects an `ExpirationManager` should be called on a DR Secondary, but we aren't there yet

if you're fairly confident i'll remove this bit! (but still update the log message)

Reply:
if it's not a batch token and not the leader node this gets forwarded. presumably dr wouldn't get into code that should be handling this: c036a1a54